env/src/main/scala/tofu/env/Env.scala

```diff
@@ -70,7 +70,7 @@ sealed trait Env[E, +A] {
     mapTask3(eb, ec)(Task.parMap3(_, _, _)(f))
 
   def zip[B](t: Env[E, B]): Env[E, (A, B)]                 = map2(t)((a, b) => (a, b))
-  def >>[B](t: Env[E, B]): Env[E, B]                       = map2(t)((_, x) => x)
+  def >>[B](t: => Env[E, B]): Env[E, B]                    = flatMap(_ => t)
```

Comment:
What is motivation of this change?
Env is referential transparent and could be passed by value just like Task

Reply:
The motivation for this PR is making the behaviour the same as Cats version. Using this operator, one could hope that it behaves exactly like Cats version, but it doesn't, so this may cause unexpected behaviour in case when used with recursive creation of argument

Reply:
Consider the following code:
```
def go: Env[Unit, Unit] = Env.unit[Unit] >> go
go.run(())
```
This code will fail with StackOverflowError, thus making it impossible to define recursive calls.
Meanwhile this code runs just fine and produces value of Env type:
```
import cats.Monad
import cats.syntax.flatMap._
import tofu.env.Env

def go[F[_]: Monad]: F[Unit] = Monad[F].unit >> go
go[Env[Unit, Unit]].run(())
```